nck/readers/dbm_reader.py

```diff
@@ -158,8 +158,11 @@ def create_and_get_query(self):
     def _wait_for_query(self, query_id):
         logging.info("waiting for query of id : {} to complete running".format(query_id))
         query_infos = self.get_query(query_id)
-        if query_infos["metadata"]["running"]:
-            raise Exception("Query still running.")
+        if query_infos["metadata"]["running"] or (
+            "googleCloudStoragePathForLatestReport" not in query_infos["metadata"].keys()
+            and "googleDrivePathForLatestReport" not in query_infos["metadata"].keys()
+        ):
+            raise ClickException("Query still running.")
```

Comment:
Do we really want a `ClickException` to be raised here? I think it might be confusing for the developer because it has nothing to do with the args he or she passed. It is all about the API. So what about a custom exception?

Reply:
I'll put an Exception like before, because here it's only used to call the retry function